.github/workflows/ci.yml

```diff
@@ -227,9 +227,27 @@ jobs:
         echo "Publishing release ${version} in Bintray"
         curl -X POST -u${BINTRAY_USERNAME}:${BINTRAY_KEY} https://api.bintray.com/content/thelastpickle/medusa-deb/cassandra-medusa/${version}/publish
 
+  publish-docker:
+    needs: [publish-debian]
+    if: github.event_name == 'release' && github.event.action == 'published'
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Publish Debian version
+      env:
+        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+      run: |
+        version=$(cat VERSION)
+        docker login -u $DOCKER_USERNAME -p $DOCKER_TOKEN
+        echo "Publishing release ${version} in Docker Hub"
+        python setup.py build
+        docker build -t thelastpickle/cassandra-medusa:${version} -f k8s/Dockerfile .
```

Comment:
Can we push the image to the k8ssandra org?

Reply:
We sure can 👍

Reply:
I'll also rename the image from `cassandra-medusa` to `medusa` in order to avoid trademark violations.